src/Plugins/RabbitMQ/RabbitMqMessageSubscriberService.cs

```diff
@@ -186,6 +214,22 @@ public void Acknowledge(MessageBase message)
             _logger.AcknowledgementSent(message.MessageId);
         }
 
+        public async Task RequeueWithDelay(MessageBase message)
+        {
+            try
+            {
+                var delay = int.Parse(_requeueDelay) * 1000;
```

Comment:
Do we want to validate `_requeueDelay` in the `ValidateConfiguration()` in case invalid input is provided?

Reply:
I have added validation to ensure that they parse to an int and that they are positive values. I have saved the values within the constructor still as they are readonly properties